repositories/droplet_repository_test.go

```diff
@@ -128,51 +128,52 @@ var _ = Describe("DropletRepository", func() {
 					Expect(k8sClient.Status().Update(testCtx, build)).To(Succeed())
 				})
 
-				It("should eventually return a record with state : \"STAGED\" set", func() {
+				It("should eventually return a droplet record with fields set to expected values", func() {
 					Eventually(func() string {
 						dropletRecord, fetchErr = dropletRepo.FetchDroplet(testCtx, client, buildGUID)
```

Comment:
It won't let me leave a comment up there, but do `fetchErr` and `dropletRecord` need to be defined up above anymore or can they just be scoped to this `It`?

Reply:
Thanks for the review. We have addressed this.